hasher-matcher-actioner/.devcontainer/Dockerfile

```diff
@@ -4,7 +4,7 @@
 # TODO: pin version later. Otherwise starting up will be too slow.
 # Note: We specify AMD64 architecture as it is currently the only one supported for this devcontainer
 # relevant for ARM architecture machines as using `python:3.8-buster` will have it try to use that instead
-FROM amd64/python:3.9-buster
+FROM python:3.9-buster
```

Comment:
Unfortunately the specificity of `FROM amd64/python:3.9-buster` is at least for now required for a successful HMA deployment. This is because various parts of the build out (specific I think we are just down to the webapp) don't have a supported runtime and error. 

If you know of a clean way to make this toggle-able I would be happy to merge it but I think we want sometime that will be closer to working by default (even if slower/not always native)

Reply:
OK, I'll follow the rest of the process through and see what else breaks, if it's not easily fixable, I'll close this.